examples/flutter_gallery/lib/welcome/steps/explore_step.dart

```diff
@@ -0,0 +1,163 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gallery/welcome/step.dart';
+
+const String _kGalleryAssetsPackage = 'flutter_gallery_assets';
+
+class PopPosition {
```

Comment:
If it's not useful anywhere else, just make this class file private. Also add a class comment for what this is for